### DISCUSSION TITLE
Why is the model not using online search?

### DISCUSSION THREAD

**Discussion by shanemcl** [perplexityai/api-discussion#48] [General]
The model has been working perfectly fine before today morning, intermittently it will work and use live data from the web in it's search, but over 60% of my api calls this morning have gotten the response that it doesn't have access to any of the data from the web and can only answer with information from it's last update in Oct 2023. To be clear, I have changed nothing in my code, and it has been working perfectly fine the last month or two, but this morning it has gone haywire ? As of this post it is working, but five minutes before it wasn't and this pattern has been repeating this morning, thank you for any info about this :)!

**Comment by shanemcl** [comment]
This issue has started occurring again :(!

**Comment by aarashy** [comment]
Hey, the `-online` models are connected to our search index, and other models are NOT. 
We'd need more details about the specific API request you're sending to debug further.
